engine/src/main/scala/cromwell/webservice/routes/wes/WesRunRoutes.scala

```diff
@@ -0,0 +1,67 @@
+package cromwell.webservice.routes.wes
+
+import akka.actor.ActorRef
+import akka.http.scaladsl.model.{HttpHeader, StatusCodes}
+import akka.http.scaladsl.server.Directives._
+import akka.http.scaladsl.server.Route
+import akka.http.scaladsl.server.directives.RouteDirectives.complete
+import akka.util.Timeout
+import WesRunRoutes._
+import akka.stream.ActorMaterializer
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.Future
+import scala.util.{Failure, Success}
+
+trait WesRunRoutes {
+  implicit def materializer: ActorMaterializer
+
+  implicit val timeout: Timeout
+
+  val serviceRegistryActor: ActorRef
+
+  lazy val wes2CromwellInterface = new Wes2CromwellInterface()
+
+  lazy val runRoutes: Route =
+    optionalHeaderValue(extractAuthorizationHeader) { authHeader =>
+      val cromwellRequestHeaders = authHeader.toList
```

Comment:
Since the headers are unused, I'd be tempted to remove the `authHeader` here and not require `headers` as a parameter to `listRuns`